This literature review seeks to add physiological detail to foundational biological concepts introduced in section {}.
In addition, I will introduce mutational signatures and give an overview of the field as applied to breast cancer, expand on the key study that led to my investigation, and ...
underpinning this project. In addition, I will describe the  the distilled executive summary included in section {}, 


### Mutational Signatures and HRDetect

As described in [12], a 2012 study analysing whole genome sequences of 21 breast cancers introduced the concept of mutational signatures. The analysis of this study demonstrated consistent mutagenic patterns across the sequenced tumour [{}]. The statistical power of thousands of accumulated somatic mutations in the tumour's sequence enables the discernment of patterns of base substitution, indel, rearrangement and copy number mutations caused by specific mutagenic processes. This enables inference of the causal mechanisms of cancer development - considered as such after validation of causation rather than association - from the sequence of the cancer itself.
SInce this intial finding, many more studies have been conducted, using larger and more diverse sets of somatic mutation data across tissue and cancer types [{}]. This has enabled the expansion of the number of identified signatures, and our understanding of the underlying biological processes at play, and the development of sophisticated tools and frameworks for mutational signature analysis. [{mutational signature review - look at reference notes}].

- difficulty in signature fititing (leads into action of HRDetect)
	- figure 3a and 3b 
- signatures present in breast cancer
- brca signatures / HR
	- figure
- HRDetect - USE MUTATIONAL SIGNATURES REVIEW
	- Overview of Action
	- combination of signatures
	- Results
	- Prognostic Benefit
	- 
	- undiagnosed patients + methylation
	- 
	- concurrence with chord 
	- critique: methylation - implications for this study
- 

### Computational Genomics and Near-Coding Regions

As described in [13], the last three decades have seen huge technological development and advances in the analysis of human genetic variation, and understaing of its role in genetic causes of disease/e

...


advances in technology, scientific understanding of 
- rise of genomics to study diseases
- pitfalls of exclusive focusing on protein-coding regions
- non-coding elements
	- figure 1 (the role of non-coding) - functional elements
	- definition of near-coding
- {CLINICAL INTERPRETATION REVIEW}
- recent demonstrations of non-coding analysis
- some particular regions of interest - regulatory + splicing
	- near -coding regions
- mechanisms

- Overview
	- 50% 
	- non-coding regions
	- recent expansion of non-coding
- key studies
	- LoF
	- MEF2C
	- Neurodevelopment disorders
- interpretation - clinical interpretation
- annotations
	- SpliceAI
	- PhyloP
	- CADD
	- UTRannotator
	- RNA Binding Protein
	- PolyA Signals