1. Davies et al., “HRDetect Is a Predictor of BRCA1 and BRCA2 Deficiency Based on Mutational Signatures.”
2. Sehgal, R.; Sheahan, K.; O'Connell, P.R.; Hanly, A.M.; Martin, S.T.; Winter, D.C. Lynch Syndrome: An Updated Review. _Genes_ **2014**, _5_, 497-507. https://doi.org/10.3390/genes5030497
3. A.K. Tiwari, H.K. Roy, H.T. Lynch, Lynch syndrome in the 21st century: clinical perspectives, _QJM: An International Journal of Medicine_, Volume 109, Issue 3, March 2016, Pages 151–158,
4. Zou, X., Koh, G.C.C., Nanda, A.S. _et al._ A systematic CRISPR screen defines mutational mechanisms underpinning signatures caused by replication errors and endogenous DNA damage. _Nat Cancer_ **2**, 643–657 (2021). https://doi.org/10.1038/s43018-021-00200-0
5. West, S. Molecular views of recombination proteins and their control. _Nat Rev Mol Cell Biol_ **4**, 435–445 (2003). https://doi.org/10.1038/nrm1127
6. Homologous recombination in DNA repair and DNA damage tolerance Xuan Li1, Wolf-Dietrich Heyer1, 2
7. Pellegrini, Luca, and Ashok Venkitaraman. "Emerging functions of BRCA2 in DNA recombination." _Trends in biochemical sciences_ 29.6 (2004): 310-316.
8. Cancer Suppression by the Chromosome Custodians, BRCA1 and BRCA2
9. https://doi.org/10.1146/annurev.genet.40.110405.090451
10. Friedman LS, Ostermeyer EA, Szabo CI, et al: Confirmation of BRCA1 by analysis of germline mutations linked to breast and ovarian cancer in ten families. Nat Genet 8:399-404, 1994
11. Wooster R, Bignell G, Lancaster J, et al: Identification of the breast cancer susceptibility gene BRCA2. Nature 378: 789-792, 1995 [Erratum: Nature 379:749, 1996]
12. Mutational signatures: emerging concepts, caveats and clinical applications
13. a brief history of human genetics
14. the role of noncoding variants in heritable disease
15. Rehm HL. Evolving health care through personal genomics. Nat Rev Genet. 2017;18:259–67 
	1. CHECK
16. Mattick JS, Dinger M, Schonrock N, Cowley M. Whole genome sequencing provides better diagnostic yield and future value than whole exome sequencing. Med J Aust. 2018;209:197–9.
	1. CHECK
17. Ellingford JM, Barton S, Bhaskar S, Williams SG, Sergouniotis PI, O’Sullivan J, et al. Whole genome sequencing increases molecular diagnostic yield compared with current diagnostic testing for inherited retinal disease. Ophthalmology. 2016;123:1143–50.
18. Gilissen C, Hehir‑Kwa JY, Thung DT, van de Vorst M, van Bon BWM, Willemsen MH, et al. Genome sequencing identifies major causes of severe intellectual disability. Nature. 2014;511:344–7
19. 100,000 Genomes Project Pilot Investigators, Smedley D, Smith KR, Martin A, Thomas EA, McDonagh EM, et al. 100,000 Genomes pilot on rare‑disease diagnosis in health care ‑ preliminary report. N Engl J Med. 2021;385:1868–80.
20. MEF2C
21. Recommendations for clinical interpretation of variants found in non-coding regions of the genome
22. Characterising the loss-of-function impact of 5’ untranslated region variants in 15,708 individuals
23.   Using high-resolution variant frequencies to empower clinical genome interpretation
24. A joint NCBI and EMBL-EBI transcript set for clinical genomics and research
25. Forrest, A.R., Kawaji, H., Rehli, M., Baillie, J.K., de Hoon, M.J., Haberle, V., Lassmann, T., Kulakovskiy, I.V., Lizio, M., Itoh, M., et al.; FANTOM Consortium and the RIKEN PMI and CLST (DGT) (2014). A promoter-level mammalian expression atlas. Nature 507, 462–470.
26. Ferreira, C. R. The burden of rare diseases. _Am. J. Med. Genet. A_. **179**, 885–892 (2019).
27. Whole-genome sequencing of patients with rare diseases in a national health system
28. A general framework for estimating the relative pathogenicity of human genetic variants
29. Detection of nonneutral substitution rates on mammalian phylogenies
30. [phyloP score (thermofisher.com)](https://ionreporter.thermofisher.com/ionreporter/help/GUID-03D1F68A-E646-4B49-AD59-AF2F51874BD2.html#:~:text=phyloP%20scores%20measure%20evolutionary%20conservation,are%20predicted%20to%20be%20conserved.)
31. Annotating high-impact 5′untranslated region variants with the UTRannotator
32. Calvo,S.E. et al. (2009) Upstream open reading frames cause widespread reduction of protein expression and are polymorphic among humans. Proc. Natl. Acad. Sci. USA, 106, 7507–7512.
33. McLaren,W. et al. (2016) The Ensembl variant effect predictor. Genome Biol., 17, 122.